.travis.yml

```
@@ -0,0 +1,43 @@
+language: bash
+sudo: false # Use container-based infrastructure
+
+os:
+  - windows
+  - linux
+  - osx
+env:
+  - PYTHON_VERSION="3.7"
+  - PYTHON_VERSION="3.8"
+
+before_install:
+  - wget https://raw.githubusercontent.com/trichter/conda4travis/latest/conda4travis.sh -O conda4travis.sh
+  - source conda4travis.sh
+
+install:
+  - conda config --add pinned_packages python=$PYTHON_VERSION
+  - conda create -n atlite python pip wheel
```

```suggestion
  - conda create -n atlite python pip wheel pytest
```
We can try this. I think I'd prefer this over an extra `environment.dev.yaml`.

---

Ah, or just add it to the `environment.yaml`.

---

> Ah, or just add it to the environment.yaml.

Yes, that was the essence of the third commit.